docs/ctrl-tab/quickey-ctrl-tab.ahk

```diff
@@ -22,6 +22,23 @@ MinUpDownTicks            := 200
 OpenedTickCount           := 0
 SawCtrlTab                := 0
 
+ExcludeTitles             := ["(Visual Studio Code|VSCodium)( - Insiders)?$"] ; VSCode / VSCodium
+
+ExcludeTitle := ExcludeTitles[1]
+For Idx, Title in ExcludeTitles {
+	If (Idx > 1) {
+		ExcludeTitle .= "|" . Title
+	}
+}
```

Comment:
I'm not really an expert in AutoHotkey.  What is this looping over?  It kind of looks like `ExcludeTitles` is a regex that might match multiple app names, but then it ends up with just a single `ExcludeTitle` string.  I guess it's building up a string of OR'd titles?

Reply:
> I guess it's building up a string of OR'd titles?

You are right. I think extending an array of OR'd regexes can be much easier than maintaining an all-in-one regex.

Reply:
NVM.  I think I understand it now.  Seems to work well!